prow/pod-utils/decorate/podspec.go

```diff
@@ -23,36 +23,211 @@ import (
 	"k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
+	"k8s.io/test-infra/prow/clonerefs"
+	"k8s.io/test-infra/prow/entrypoint"
+	"k8s.io/test-infra/prow/gcsupload"
+	"k8s.io/test-infra/prow/initupload"
 	"k8s.io/test-infra/prow/kube"
+	"k8s.io/test-infra/prow/pod-utils/clone"
 	"k8s.io/test-infra/prow/pod-utils/downwardapi"
+	"k8s.io/test-infra/prow/pod-utils/wrapper"
+	"k8s.io/test-infra/prow/sidecar"
 )
 
+const (
+	LogMountName            = "logs"
+	LogMountPath            = "/logs"
+	ArtifactsEnv            = "ARTIFACTS"
+	ArtifactsPath           = LogMountPath + "/artifacts"
+	CodeMountName           = "code"
+	CodeMountPath           = "/home/prow/go"
```

Comment:
Will this be hard to change later? I don't like that we use a go-specific path.

Reply:
Again this is literally an ephemeral path that can be mounted anywhere and has no meaning outside of the container. It does not exist until the container is started and the mount is created. It should not impact anything anywhere. Nobody should be aware of this path or use it for anything.

Reply:
And allowing someone to configure it to point somewhere misleads them into thinking that we will somehow use whatever may exist at that path already instead of clobbering it with the volume mount.

Reply:
I wouldnt want to make this configurable. Do you need it to be go-specific?